native/mod_proxy_cluster/mod_proxy_cluster.c

```diff
@@ -297,17 +377,22 @@ static apr_status_t create_worker(proxy_server_conf *conf, proxy_balancer *balan
                     if (proxyhctemplate != NULL) {
                         add_hcheck(server, conf, worker);
                     }
+                    ap_log_error(APLOG_MARK, APLOG_DEBUG, 0, server,
+                                 "Created: REUSING %s (scheme: %s hostname %s hostname_ex %s port %d route %s name %s) cleaning...",
+                                  url, worker->s->scheme, worker->s->hostname, worker->s->hostname_ex, worker->s->port, worker->s->route, worker->s->name_ex );
                 }
                 return APR_SUCCESS; /* Done Already existing */
             } else {
                 ap_log_error(APLOG_MARK, APLOG_DEBUG, 0, server,
-                             "Created: can't reuse worker as it for %s cleaning...", url);
+                             "Created: can't reuse worker as it for %s (scheme: %s hostname %s hostname_ex %s port %d route %s name %s) cleaning...",
```

Comment:
Isn't there a typo? `... worker as it for ...` -> `... worker as is for ...`

Reply:
fixed